downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -88,8 +88,20 @@ To view bug fixes, see the GitHub milestone for Swan Lake 2201.6.0 of the reposi
 
 ### New features
 
+#### Language Server
+
+- Added completions for the `group by` clause.
+- Added inlay hint support for function call expressions and method call expressions to provide information about parameters.
+
 ### Improvements
 
+#### Language Server
+
+- Removed service template initialization from the lightweight mode.
+- Improved completions in the client resource access action node context.
+- Improved the main function completion item.
+- improved completions in the named argument context.
+
 ### Bug fixes
```

Comment:
Let's add a link to the LS bug fixes as well.

Eg: In the previous release note
![image](https://github.com/ballerina-platform/ballerina-dev-website/assets/35211477/e7e44a2f-5e58-4c8a-852b-b0b99770d407)

Reply:
There is a link already